pvlib/ivtools/fit_sde.py

```diff
@@ -459,5 +461,13 @@ def _cocontent_regress(v, i, voc, isc, cci):
 
     # translate from coefficients in rotated space (gamma) to coefficients in
     # original coordinates (beta)
-    beta = np.linalg.lstsq(np.dot(mb, ma), gamma[0:5], rcond=None)[0]
+    beta = np.linalg.lstsq(np.dot(mb, ma), gamma[0:5], rcond=rcond)[0]
     return beta
+
+
+def _set_rcond():
+    # handles change in rcond in numpy.linalg.lstsq
+    if np.__version__ < '1.14':
```

Comment:
This is a risky pattern -- `'1.9' < '1.14'` is False, though since pvlib requires numpy 1.12+, maybe this won't be a problem until numpy version `1.100`...

Reply:
Rats. Hopefully the intent is clear - any suggestions here?

Reply:
Ideally there would be something like `sys.version_info >= (3, 5)`, but there's nothing in `np.version` that isn't a string.  Off the top of my head I don't have anything better than a DIY approach:
```
In [53]: v = "1.13.9".split("."); (int(v[0]), int(v[1])) < (1, 14)
Out[53]: True

In [54]: v = "1.14.1".split("."); (int(v[0]), int(v[1])) < (1, 14)
Out[54]: False
```

Reply:
Thanks! Perfectly fine for a workaround that will go away when we pin to numpy 1.14 or later.

Reply:
we use `pkg_resources.parse_version` in [`conftest.py`](https://github.com/pvlib/pvlib-python/blob/master/pvlib/test/conftest.py#L7) but it appears that [`packaging.version.parse`](https://packaging.pypa.io/en/latest/version/#packaging.version.parse) is now [preferred](https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python).

Reply:
Weird - wonder how my response comment ended up as a review....